<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QDialog" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEPP-SIGNUP </string>
  </property>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>132</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usr">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_verpwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="VerifyPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Retype Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>5</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_pic</zorder>
  <zorder>Submit</zorder>
  <zorder>Username</zorder>
  <zorder>NewPassword</zorder>
  <zorder>lineEdit_usr</zorder>
  <zorder>lineEdit_verpwd</zorder>
  <zorder>VerifyPassword</zorder>
  <zorder>lineEdit_pwd</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
